app/services/submission_service.rb

```diff
@@ -18,18 +18,22 @@ def create_submission(
         submission_params.delete(:edition_size_formatted) if submission_params[
         :edition_size_formatted
       ]
-      user = User.find_by(gravity_user_id: gravity_user_id) if gravity_user_id
-        .present?
 
-      create_params = submission_params.merge(user_id: user&.id)
+      create_params = submission_params.clone
```

Comment:
And what is the meaning of renaming?

Reply:
I suppose to prevent mutation of submission_params, but since it happened above I don't have the exact answer.